modules/ROOT/pages/dw-multipart-functions-form.adoc

```diff
@@ -25,6 +25,103 @@ This example creates a `Multipart` data structure that contains parts, which
 are described in examples for the `field` function. For additional uses of
 `form`, see examples in the Multipart `file` and `field` documentation.
 
+You can also manually construct a data structure compatible with `multipart/form-data` output. 
+
+In this example, the first part uses the `Multipart::file()` function to import an external file named `orders.xml` file. In this case, the path is internal to the Mule application, where the file is located in the `src/main/resources` folder. See the `file` docs for more details on this example. 
+
+The second part uses the `Multipart::field()` function. It uses the version of the function where the field names are included in the input parameter as an object of key/value pairs, so the keys can be arranged in any order. In this example, and additional key named `extra` is included, and it is ignored by the `Multipart::field()` function. This part also does not specify the optional `fileName` parameter. When specified, the `fileName` is part of the `Content-Distribution` element of the part. The `mime` field is also optional. When included, this sets the `Content-Type` element to the `mime` value. In this case the `Content-Type` is set to `text/plain`. 
+
+The third part uses the more compact version of the `Multipart::field()` function where the required and optional parameters are entered, in the correct order, as input parameters. The first three parameters `name`, `value`, and `mime` are required. The `fileName` parameters is optional, and should only be used if the content came from a file or should be written to a file. In this version, the `mime` parameter is output as the `Content-Type` element, and the `fileName` is output as the `filename` parameter of the `Content-Distribution` element. 
+
+==== Source
+
+[source,DataWeave,linenums]
+----
+%dw 2.0
+import dw::module::Multipart
+output multipart/form-data
+
+var myOrders = "./orders.xml"
+var fileArgs = { name: "file", path: myOrders, mime: "application/xml", fileName: "myorders.xml"}
+var fieldArgs = {name:"userName",value: "Annie Point", mime: "text/plain", extra: "ignored"}
+---
+Multipart::form([
+  Multipart::file(fileArgs),
+  Multipart::field(fieldArgs),
+  Multipart::field("myJson", {"user": "Annie Point"}, "application/json", "userInfo.json")
+])
+----
+
+Each part can be coerced to the `Multipart::MultipartPart` type, and the whole form can be coerced to the `Multipart::Multipart` type. 
+
+So this source code gives equivalent output:
+
+==== Source
```

Comment:
I don't wouldn't add this. It is not invalid or wrong, it is just that is too much info for something that is not that relevant IMO

Reply:
Mariano is referring to everything from here down:

> Each part can be coerced to the `Multipart::MultipartPart` type, and the whole form can be coerced to the `Multipart::Multipart` type. 
> 
> So this source code gives equivalent output: